specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-05-01/machineLearningServices.json

```diff
@@ -2177,6 +2966,264 @@
+        "properties": {
+          "$ref": "#/definitions/WorkstationProperties",
+          "x-ms-client-flatten": true,
+          "title": "Workstation update properties",
+          "description": "The properties for a workstation"
+        }
+      }
+    },
+    "WorkstationProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state of the workstation, which only appears in the response."
+        },
+        "vmSize": {
+          "type": "string",
+          "title": "VM size.",
+          "description": "The size of the virtual machine for the workstation. For information about available VM sizes from the Virtual Machines Marketplace, see Sizes for Virtual Machines."
+        },
+        "osProfile": {
+          "$ref": "#/definitions/UserAccountCredentials",
+          "title": "OS profile",
+          "description": "The OS profile of a workstation"
+        },
+        "networkProfile": {
+          "$ref": "#/definitions/NetworkProfile",
+          "title": "Network profile",
+          "description": "The network profile of a workstation"
+        }
+      },
+      "required": [
+        "vmSize",
+        "osProfile"
+      ],
+      "description": "The properties of a workstation."
+    },
+    "WorkstationUpdateProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state, which only appears in the response."
+        },
+        "vmSize": {
+          "type": "string",
+          "title": "VM size.",
+          "description": "The size of the virtual machine for the workstation. For information about available VM sizes from the Virtual Machines Marketplace, see Sizes for Virtual Machines."
+        }
+      },
+      "description": "The update properties of a workstation."
+    },
+    "UpdateResource": {
+      "description": "The Update Resource model definition.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "WorkstationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workstation"
+          },
+          "description": "The collection of workstations."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The continuation token."
+        }
+      },
+      "description": "Values returned by the workstation list operation."
+    },
+    "NetworkProfile": {
+      "description": "Network profile for the workstation",
+      "properties": {
+        "subnetId": {
+          "description": "Subnet ID",

```

Comment:
I assume a linked access check will be defined for this?

Reply:
Yes, we will add a linked access check for this in our manifest file.